src/libraries/illink-oob.targets

```diff
@@ -0,0 +1,63 @@
+<Project>
+
+  <Target Name="ILLinkTrimOOBAssemblies"
+          AfterTargets="Build"
+          DependsOnTargets="PrepareForAssembliesTrim">
+
+    <Message Text="Trimming $(PackageRID) OOB assemblies with ILLinker..." Importance="high" />
+
+    <PropertyGroup>
+      <LibrariesTrimmedOOBArtifactsPath>$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'ILLinkTrimAssembly', '$(BuildSettings)', 'trimmed-oobs'))</LibrariesTrimmedOOBArtifactsPath>
```

Comment:
Can we use `$(ILLinkTrimAssemblyArtifactsRootDir)` here?

Reply:
Yes, so I'll go ahead and use it, that said it will introduce a order dependency that will require illink.targets to get imported always before these other target files, so in order to not create that order dependency, I will switch to use that property but also add a check to default it in case it is not set already. That way, if a refactoring happens in the future we won't break this by mistake, and it would be easier to maintain.

Reply:
There's no ordering issue because this happens in a `Target`, and the setting of that Property happens in a global `PropertyGroup`. So no matter which file gets included first, the Property will be set first.